### PR TITLE
Chore/auth testing

### PR DESCRIPTION
**Pull request by lfbrehm**
This PR contains bug fixes for:
- Authorization logic
- Naming validation for resources
- License defaults
- `CreateHooks` are now triggered on finish for objects
- Timestamps for hooks are now parsed from seconds

### CODE REVIEW

File: src/auth/structs.rs [hunk 1]
```diff
@@ -68,7 +68,7 @@ impl Context {
 impl Default for Context {
     fn default() -> Self {
         Self {
-            variant: ContextVariant::Activated,
```
Comment (St4NNi) [inline src/auth/structs.rs:71]: I think default should be activated and notactivated the "special" case.
Just to be sure that as a bare minimum the user is activated

File: src/grpc/search.rs [hunk 2]
```diff
@@ -182,7 +174,8 @@ impl SearchService for SearchServiceImpl {
             match object_plus.object.data_class {
                 DataClass::PUBLIC => {}
                 DataClass::PRIVATE => {
-                    // TODO: Redacted object operations
+                    // TODO: SPECIFIC private operations OTHER THAN strip labels
+                    // -> maybe endpoint redaction?
```
Comment (das-Abroxas) [inline src/grpc/search.rs:178]: Also the field `created_by` could be emptied even if it is 'just' an id.

File: src/grpc/search.rs [hunk 3]
```diff
@@ -316,7 +309,8 @@ impl SearchService for SearchServiceImpl {
                 match object_plus.object.data_class {
                     DataClass::PUBLIC => {}
                     DataClass::PRIVATE => {
-                        //TODO: Redacted object
+                        // TODO: SPECIFIC private operations OTHER THAN strip labels
+                        // -> maybe endpoint redaction?
```
Comment (das-Abroxas) [inline src/grpc/search.rs:313]: Same as above.

**Review by das-Abroxas — APPROVED** [review]
Overall looks good to me.

**Review by St4NNi — COMMENTED** [review]
Just a minor change!

**Pull request merged**